src/drivers/uefi/lv_uefi_context.c

```diff
@@ -0,0 +1,170 @@
+/**
```

Comment:
It could be placed in the [stdlib](https://github.com/lvgl/lvgl/tree/master/src/stdlib) folder.

Reply:
You mean the memory functions?

Reply:
Yes

Reply:
It would be the only stdlib implementation without support for all 3 method groups (mem, sprintf and string), so i would not create a new LV_STDLIB_UEFI switch. I suggest to enable the file with LV_USE_STDLIB_MALLOC == LV_STDLIB_CUSTOM && LV_USE_UEFI.

Reply:
Okay, let's do it as you have suggested.